src/FlexGrid/FlexColumn/index.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export default function Col({ children, col, offset, style }) {
+  const classCol = col && `col-${col}`;
```

Comment:
just call this column

Reply:
so is as you said that i prefix with class because the names are taken from the props.
I will leave it like it is if thats fine